gradient/api_sdk/clients/experiment_client.py

```diff
@@ -328,23 +409,52 @@ def run_multi_node(
     def start(self, experiment_id):
         """Start existing experiment
 
-        :param str experiment_id:
+        *EXAMPLE*::
```

Comment:
can you add a brief description here, along the lines of `Start a created Experiment that has not run.`